### ISSUE TITLE
Investigation and learning JAWS for NMDC use case

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/issues#14]
**Describe the solution you'd like**
Learning the JAWS code and how the NMDC workflows can use JAWS to run more efficiently.  

**Describe alternatives you've considered**
Continue as is.

**Acceptance Criteria**
Create a checklist or scenario-based acceptance criteria, from the users perspective, that answers the following: 

Who will use this feature/enhancement? To start Mark Flynn will be training and learning about JAWS. 
When will they use it? Testing and seeing if the NMDC workflows will work effectively using JAWS.
How will they use it? To use the infrastructure created by JAWS so we don't duplicate work for NMDC. 
How will they test it to make sure it's working? Running NMDC workflows
Is the request achievable? During one sprint? Learning will start this sprint, likely will continue.
What is your definition of done for this request? For Mark to document his learning and determine the best way to leverage JAWS for NMDC workflows. Future issues can cover implementation, Edge use case, etc.

**Comment by ssarrafan** [comment]
Mark has met with the JAWS team and is getting training on JAWS.  I'm closing this issue for this sprint. I'll open a new issue for the next sprint when the NMDC JAWS instance is up so that Mark can start testing it out.  
@mflynn-lanl let me know if there should be any additional NMDC JAWS related issues.
